### PR TITLE
Readme update for Apiary-Replication components

### PR DESCRIPTION
**Pull request by abhimanyugupta07**
Added apiary-replication, apiary-extensions-terraform, apiary-shuntingyard-docker to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,22 @@ Apiary provides modules which can be combined to create a federated cloud data l
...
 Apiary consists of the following components which are managed in separate git repositories:
 * [Apiary Authorization](https://github.com/ExpediaGroup/apiary-authorization)
 * [Apiary Data Lake](https://github.com/ExpediaGroup/apiary-data-lake)
-* [Apiary Extensions](https://github.com/ExpediaGroup/apiary-extensions)
+* [Apiary Replication](https://github.com/ExpediaGroup/apiary-replication)
 * [Apiary Federation](https://github.com/ExpediaGroup/apiary-federation)
 * [Apiary Metastore Docker](https://github.com/ExpediaGroup/apiary-metastore-docker)
 * [Apiary Ranger Docker](https://github.com/ExpediaGroup/apiary-ranger-docker)
 * [Apiary Waggle Dance Docker](https://github.com/ExpediaGroup/apiary-waggledance-docker)
+* [Apiary Shunting Yard Docker](https://github.com/ExpediaGroup/apiary-shuntingyard-docker)
```
Comment (max-jacobs) [inline README.md:23]: Looks as though these were in alphabetical order, perhaps we should keep them that way for readability?

Comment (abhimanyugupta07) [reply README.md:23]: They do ! But I feel that might be a coincidence. I felt the intention was to arrange it as follows:

main components
docker images of those components.

I just combined the extensions and its terraform at the end together. Happy to make it alphabetical if thats how we want to keep.

Comment (pradeepbhadani) [reply README.md:23]: yes they are in alphabetical order.

Comment (abhimanyugupta07) [reply README.md:23]: Alright. done.

**Review by massdosage — APPROVED** [review]

**Review by max-jacobs — APPROVED** [review]

**Pull request merged**